lightning/src/chain/package.rs

```diff
@@ -374,6 +387,17 @@ impl PackageSolvingData {
 			_ => { panic!("API Error!"); }
 		}
 	}
+	fn absolute_tx_timelock(&self, input_height: u32) -> u32 {
+		let amt = match self {
```

Comment:
I think input_height should be called "output_conf_height". I concede that height original isn't really clear, but it avoids at least letting think that this timelock are only CSV ones.

Maybe rename `amt` to `absolute_height`?

And precise the +1 is to follow mempool rules of evaluating timelocks, otherwise a newcomer can be easily confused on this.